.github/workflows/build_wheels.yml

```diff
@@ -24,32 +24,33 @@ jobs:
       - name: Build sdist
         run: pipx run build --sdist
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
+          name: sdist
           path: dist/*.tar.gz
 
-  choose_architectures:
-    name: Decide which architectures to build wheels for
+  choose_wheel_types:
+    name: Decide which wheel types to build
     runs-on: ubuntu-latest
     steps:
-      - id: x86_64
-        run: echo "cibw_arch=x86_64" >> $GITHUB_OUTPUT
-      - id: aarch64
+      - id: manylinux_x86_64
+        run: echo "wheel_types=manylinux_x86_64" >> $GITHUB_OUTPUT
```

Comment:
Why do you use an underscore (`_`) instead of a dash (`-`) for the separation between `manylinux` and `x86_64`? I would prefer a dash if possible, to make it clear that `x86_64` is really a single unit.

Reply:
This gets expanded in `CIBW_BUILD: "cp3{7..12}-${{ matrix.wheel_type }}"` and cibuildwheel uses a `_` instead of a `-` there.  See https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip for examples, but we need that to expand to (for instance) `cp312-musllinux_x86_64`